README.md

```diff
@@ -69,7 +84,8 @@ git remote add staging git@heroku.com:artsy-metaphysics-staging.git
 Then promote using the command:
 
 ```sh
-$ heroku pipelines:promote -r staging
+$ yarn promote:production
```

Comment:
This is the only real change; the rest is Prettier formatting markdown.

Reply:
good good